modules/doc/content/framework_development/authorship_guidelines.md

```diff
@@ -0,0 +1,73 @@
+of these articles. These articles are published on a regular basis.
+
+## MOOSE framework paper authorship guidelines
+
+When citing MOOSE, whether when using it, using a derived application, or just referring to
+the MOOSE project, this is the article that should be cited. It is featured in the [Citing.md] page.
+This paper is regularly replaced with either a new paper or an "update" paper. "Update" papers
+refer to an update of the software and the paper, and can have a different authorship list as the original
+paper.
+
+In order to best acknowledge the current contributors to the framework, we strive to keep the authorship
+of the MOOSE framework paper representative. To do that, we publish a summary of the new developments in the framework
+and use the list of the principal contributors since the previous paper as the list of authors for the new paper.
+
+The contributors are listed in two categories.
+
+- Semi-permanent contributors are included in every new MOOSE framework paper by default. These contributors
+  have written a very large part of the MOOSE framework and must be credited, at least until their contributions
+  are overtaken by others.
+
+The current criteria for semi-permanent authorship are listed below:
+
+- Top 10 line of codes in the `framework/` and other utility folders or
+- Top 10 commits in the `framework/` folder
+
+
+!alert note
+While it seems poor practice to recognize contribution by LOC or commit numbers, this criteria did not
+exist when these contributions were made, so the metric did not influence the coding style of these authors.
+These criteria may be reformulated in the future in terms of features.
+
+
+- Transient contributors are included in the next MOOSE framework paper for the period(s) in which they contributed.
+  These contributors have played an important role in MOOSE development since the previous paper was published.
+
+
+The current guidelines for transient authorship are listed below:
+
+- 1 major contribution or
+- 4 minor contributions and significant involvement in MOOSE-related activities or
+- 10 fixup contributions and significant involvement in MOOSE-related activities
+
+
+Contributions are only tallied for the *framework and supporting utilities*. Modules are deliberately excluded
+as developers of a module should publish a module paper and report on new developments outside of the MOOSE framework paper.
+MOOSE-related activities are developments in the MOOSE modules and MOOSE applications.
+
+A list of major, minor and fixup example contributions has been developed as guidelines for
+classification and can be shared upon request. The classification of contributions is decided when
+gathering the list of authors for the next paper.
+
+!alert note
+Contributors who do not meet the criteria for authorship in the paper will see their contributions from
+the past period considered for the next paper.
```

Comment:
The wording here is a bit unclear. Does this mean that a contributor who doesn't meet the guidelines for the new update paper will have their contributions from the same period considered for the next paper (on top of new contributions in the intervening period)?

Consider revising.

Reply:
yes that would be my intention. 2 over 1 period then 2 over the next makes 4 for the next